File: python/Template/complete/HaikuOne.py
from Template import Template

class HaikuOne(Template):
    '''
    implements the methods from superclass Template to write a haiku
    Winter seclusion
    Listening, that evening
    To the rain in the mountain
    '''

    def write_first_line(self):
        return "Winter seclusion"

    def write_second_line(self):
        return "Listening, that evening"

    def write_third_line(self):
        return "To the rain in the mountain"

File: java/Template/complete/HaikuOne.java
public class HaikuOne extends Template{
  /*
  Overrides the abstract methods from superclass Template to write a haiku
  Winter seclusion
  Listening, that evening
  To the rain in the mountain
  */

  @Override
  public String write_first_line(){
    return "Winter seclusion";
  }

  @Override
  public String write_second_line(){
    return "Listening, that evening";
  }

  @Override
  public String write_third_line(){
    return "To the rain in the mountain";
  }
}
